xds/src/main/java/io/grpc/xds/internal/sds/SdsProtocolNegotiators.java

```diff
@@ -48,7 +45,6 @@
  * Provides client and server side gRPC {@link ProtocolNegotiator}s that use SDS to provide the SSL
  * context.
  */
-@Internal
 public final class SdsProtocolNegotiators {
```

Comment:
This one doesn't need to be public any more, right?

Reply:
That's right. Thanks for the catch. Done.